packages/web-components/fast-foundation/src/listbox-option/listbox-option.ts

```diff
@@ -217,7 +217,7 @@ export class ListboxOption extends FoundationElement {
         Observable.notify(this, "value");
     }
 
-    public get value(): string {
```

Comment:
@radium-v what is the purpose of accepting `unknown` here?

Reply:
Any external source can set an option's value with `option.value = <any value>` but it should always be handled as a `string`. Setting the type to `string` alone only gives us the sense that only `string` type values will ever be provided, which isn't the case. #5925 provides some more context.

Reply:
Yeah okay that makes sense. `value` being an unknown type feels pretty akward to me, from the perspective of a TypeScript users. I understand that the bug in that example can happen in JavaScript, but a TypeScript compiler would have caught that particular issue compile-time.

I wonder if the right fix here is to change the value setter back to `set value(next: string)`. This will provide a better author experience for TypeScript users. In that setter we can still do any conversion we need to for folks using JavaScript and those circumventing the type system. What are your thoughts on that?

Reply:
I agree with @nicholasrice on this.

Reply:
I'm not sure of the fix? Reverting the change would bring back the typescript error in 4.0.5. Kindly suggest what to do.

Reply:
I believe we were proposing that both the get and set have a simple type of `string`. I needed, the setter/getter can guard/check for `undefined` internally.